digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -213,39 +221,75 @@ public TDJobOperator submitNewJobWithRetry(Submitter submitter)
         }
     }
 
-    public TDJobOperator submitExportJob(TDExportJobRequest request)
+    public TDJobOperator newJobOperator(String jobId)
     {
-        // TODO retry with an unique id and ignore conflict
-        return newJobOperator(client.submitExportJob(request));
+        return new TDJobOperator(client, jobId);
     }
 
-    public TDJobOperator submitPartialDeleteJob(String table, Instant from, Instant to)
+    /**
+     * Run a TD job in a polling non-blocking fashion. Throws TaskExecutionException.ofNextPolling with the passed in state until the job is done.
+     */
+    public TDJobOperator runJob(Config state, String key, JobStarter starter)
     {
-        // TODO retry with an unique id and ignore conflict
-        return newJobOperator(client.partialDelete(database, table, from.getEpochSecond(), to.getEpochSecond()).getJobId());
-    }
+        ///////////////////////////////////////////////////////////////////////////////////////////
+        // TODO: remove this migration code
+        if (state.has("jobId")) {
+            Config jobState = state.getNestedOrSetEmpty(key);
+            if (!jobState.isEmpty()) {
+                throw new AssertionError();
+            }
+            jobState.setOptional("jobId", state.getOptional("jobId", String.class));
```

Comment:
[minor] These codes are a bit confusing because it depends on the implementation that `state.getNestedOrSetEmpty()` returns a reference not a copy.